commands/label.go

```diff
@@ -211,6 +211,7 @@ func RunLabelList(c *core.CommandConfig) error {
 }
 
 func RunLabelGet(c *core.CommandConfig) error {
+	c.Printer.Info("Getting label...")
```

Comment:
Would it make more sense details like: getting label with $label_key, value $label_Value for $resource_type... ? Same for urn and remove commands.

Reply:
done